cmake/compiler_flags_Intel_C.cmake

```diff
@@ -1,8 +1,16 @@
 # Intel C
-set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -sox -traceback")
-
-set( CMAKE_C_FLAGS_RELEASE "-qno-opt-dynamic-align -O2 -debug minimal -fp-model source")
-
-set( CMAKE_C_FLAGS_DEBUG "-O0 -g -ftrapuv")
+if(CMAKE_C_COMPILER_ID STREQUAL "Intel")
```

Comment:
This file is for providing flags for the `Intel` `C` compiler.
Please create a file `compiler_flags_IntelLLVM_C.cmake` for the `IntelLLVM` `C` family.